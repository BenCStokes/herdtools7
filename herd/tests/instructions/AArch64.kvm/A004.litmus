AArch64 A004
Variant=fatal
{
[PTE(x)]=(oa:PA(x),valid:0);
0:X1=x;
}
 P0          ;
L0:          ;
 LDR W0,[X1] ;
 forall(fault(P0:L0,x))
