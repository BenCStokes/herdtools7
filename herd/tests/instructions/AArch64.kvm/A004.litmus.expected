Test A004 Required
States 1
 Fault(P0:L0,x,MMU:Translation);
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (fault(P0:L0,x))
Observation A004 Always 1 0
Hash=abfed1dff47d259c77dbd4bc5cef0cc5

